Configuration/TCA/Overrides/pages.php

```diff
@@ -10,18 +10,34 @@
 (function ($extensionKey = 'form_custom_templates', $table='pages') {
     // Add page type
     $emailDoktype = (string)GeneralUtility::makeInstance(ExtensionConfiguration::class)->get('form_custom_templates', 'doktype');
-    ExtensionManagementUtility::addTcaSelectItem(
-        $table,
-        'doktype',
-        [
-            'LLL:EXT:' . $extensionKey . '/Resources/Private/Language/Database:form_custom_templates.pageType',
-            $emailDoktype,
-            'apps-pagetree-page-email',
-            'special',
-        ],
-        '254',
-        'before'
-    );
+
+    if ((\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class))->getMajorVersion() < 12) {
```

Comment:
```suggestion
    if ((GeneralUtility::makeInstance(Typo3Version::class))->getMajorVersion() < 12) {
```

Maybe remove the unnecessary qualifier?